9_distributed_training_kfto.ipynb

```diff
@@ -145,15 +150,21 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "### Create PyTorchJob\n",
+    "### Create a PyTorchJob\n",
+    "\n",
+    "Use the Training Operator SDK client to submit a PyTorchJob.\n",
+    "\n",
+    "The model training script is imported from the `kfto-scripts` folder.\n",
+    "\n",
+    "The model training script loads and distributes the training data set among nodes, performs distributed training, evaluates by using the test data set, and exports the trained model to ONNX format and uploads it to the S3 bucket that is specified in the provided connection.\n",
     "\n",
-    "Submit PyTorchJob using Training Operator SDK client.\n",
+    "*Important note:* If the Kueue component is enabled in OpenShift AI, you must make the following edits to the script in the next Python cell: \n",
     "\n",
-    "Training script is imported from `kfto-scripts` folder.\n",
+    "* Create all Kueue related resources (`ResourceFlavor`, `ClusterQueue`, and `LocalQueue`).\n",
```

Comment:
Based on the sentence above, it sounds like creating Kueue resources is achieved by editing provided script.
The Kueue resources however needs to be created manually on cluster, as described in https://github.com/rh-aiservices-bu/fraud-detection/blob/805b53bac755ae2bd3478a6299873e78ae243e00/workshop/docs/modules/ROOT/pages/setting-up-kueue-resources.adoc

Reply:
Thank you @sutaakar. 

I’ve updated this section in my latest commit. Please verify that my latest changes area correct. I’ve assumed that they would have done the setup steps.  Is there any use case when they would run this notebook without having done those setup steps?

Reply:
It is not needed for sandbox environment for now - as Kueue is not activated there yet

Reply:
Otherwise looks ok
Thinking if to mention that it is not needed for sandbox, though that would be a duplication of info from Kueue setup page.